specifyweb/frontend/js_src/lib/components/TreeView/Row.tsx

```diff
@@ -168,16 +194,28 @@ export function TreeRow({
                     ? 'outline outline-1 outline-blue-500'
                     : ''
                 }
+                ${
```

Comment:
you should modify the ternary above rather than add a new ternary here - the code above already checks if tree is focused or is in action - you should just modify that code